mlrun/projects/project.py

```diff
@@ -3797,6 +3798,62 @@ def delete_api_gateway(
 
         mlrun.db.get_run_db().delete_api_gateway(name=name, project=self.name)
 
+    def store_alert_config(self, alert_data: AlertConfig, alert_name=None):
+        """
+        Create/modify an alert.
+        :param alert_data: the data of the alert.
```

Comment:
:param alert_data: -- The data of the alert.
        :param alert_name: -- The name of the alert.